notebooks/Benchmark_notebook.ipynb

```diff
@@ -197,23 +217,26 @@
     "    \"\"\"Evaluates given integrator and returns result and runtime\n",
     "    \"\"\"\n",
     "    start = perf_counter()\n",
-    "    result = integrator(fn,dim,N,domain)\n",
+    "    result, grid_runtime, fn_runtime = integrator(fn,dim,N,domain)\n",
     "    time = perf_counter() - start\n",
-    "    return result,time\n",
+    "    return result, grid_runtime, fn_runtime, time -(grid_runtime + fn_runtime)\n",
```

Comment:
This is a good place to actually return runtime_f. It may be good though to make a deepcopy of it to avoid accidental modification (global variable and all...)